### DISCUSSION TITLE
Is it possible to use a local file for the redirect page?

### DISCUSSION THREAD

**Discussion by simon-friedberger** [proginosko/LeechBlockNG#415] [Q&A]
I tried and it isn't working but maybe I have the path formatting wrong. I copied it from the address bar.
In case it's not clear, I have a local image on my disk that I would like to display instead of blocked pages.
I am on OSX and the path I tried looks like this: `file:///Users/simon/foo_23234-2048x1367.jpg`

**Comment by proginosko** [comment]
Firefox's security model doesn't allow extensions to open local files (unless they're packed within the extension itself). So you'll have to put the image on a web server instead and use HTTP access. I believe some users have made it work with a cloud drive like Dropbox or OneDrive.

**Comment by waveringheights** [comment]
@proginosko Is it still the case that local files can't be used? I was trying to find a way to do this and found [this documentation](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Working_with_files#open_files_in_an_extension_using_a_file_picker).

**Reply by proginosko** [reply]
Looks like that has to do with an extension loading/accessing local files, not opening local files in the browser, which is the issue here. Sorry!

**Reply by proginosko** [reply]
Actually, I just saw that Firefox has added a permission "Access local files on your computer" for extensions. You might try setting that permission and see if you can use a local file as the blocking page.
